### ISSUE TITLE
Search test in fundamental should not require backlink

### ISSUE THREAD

**Issue by GeorgeKerscher** [daisy/transitiontoepub#44]
the Fundamental Basic test book we have nav-410 Perform a search, review the search results test. When you reach the Taj Mahal, to pass the test you must follow a backlink. Normally when conducting a search, there will not be a backlink. I propose we eliminate the requirement to follow the backlink. This would make the test more atomic. We test internal links in the non-visual reading book. 

We may want to put the internal links into the Basic title or it would not get tested when not using a screen reader.

**Comment by clapierre** [comment]
I agree with this improvement as you could fail the back linking but the search worked.
